plugins/node/opentelemetry-plugin-mongoose/README.md

```diff
@@ -0,0 +1,23 @@
+# OpenTelemetry Mongoose Plugin
```

Comment:
This need to match the style of ours readmes, example with mongodb: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/master/plugins/node/opentelemetry-plugin-mongodb/README.md